backend/src/agents/tools/browser_tools.py

```
@@ -0,0 +1,516 @@
+from crewai.tools import BaseTool
+from pydantic import BaseModel, Field
+from playwright.async_api import Page, Error as PlaywrightError
+import asyncio
+from bs4 import BeautifulSoup, NavigableString, Comment
+import re
+from typing import Dict, List, Optional,Literal
```

Ruff (0.12.2)

`typing.Literal` imported but unused

Remove unused import: `typing.Literal`

(F401)